src/rvpt/bvh.cpp

```diff
@@ -0,0 +1,20 @@
+﻿//
```

Comment:
I should really go and remove these.

Reply:
Eh, it's technically correct so it's fine.